lib/platform/bitbucket-server/index.js

```diff
@@ -661,6 +687,7 @@ async function createPr(
     toRef: {
       id: `refs/heads/${base}`,
     },
+    reviewers,
```

Comment:
Is "reviewers" a valid field when creating a PR? In addReviewers we are using "participants" - are they different things, and do we need to change anything?

Reply:
[pr docu](https://docs.atlassian.com/bitbucket-server/rest/5.0.0/bitbucket-rest.html#idm45659054371472)
[participants docu](https://docs.atlassian.com/bitbucket-server/rest/5.0.0/bitbucket-rest.html#idm45659055527056)

`reviewers` is a valid field.

I will update the `addReviewers` to use the [put pr](https://docs.atlassian.com/bitbucket-server/rest/5.0.0/bitbucket-rest.html#idm45659054260896) request to add reviewers.